<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recorder</class>
 <widget class="QDialog" name="Recorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VLC Recorder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vlc-record.qrc">
    <normaloff>:/app/tv</normaloff>:/app/tv</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Recorder {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eee, 	stop:0.7 grey, stop:1 #dedede);
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="40,60">
     <item>
      <layout class="QVBoxLayout" name="vLayoutChannels" stretch="6,4">
       <item>
        <widget class="QGroupBox" name="groupChannels">
         <property name="title">
          <string> Channels </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,60">
          <item>
           <widget class="QComboBox" name="cbxChannelGroup">
            <property name="whatsThis">
             <string>Channel Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string>Channel List</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupEpgShort">
         <property name="title">
          <string> Now Playing </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="CEpgBrowser" name="textEpgShort">
            <property name="whatsThis">
             <string>Short EPG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>10</height>
             </size>
            </property>
            <property name="whatsThis">
             <string>Time Progress for played programm</string>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupEPG">
       <property name="title">
        <string> EPG - Electronic Program Guide </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,10,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="hLayoutEpgNavi">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frameEpgCtrl">
          <property name="styleSheet">
           <string notr="true">QFrame#frameEpgCtrl {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eee, 	stop:0.7 grey, stop:1 #dedede);
};</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="margin">
            <number>4</number>
           </property>
           <item>
            <widget class="QFrame" name="frameChan">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="labChanIcon">
                <property name="minimumSize">
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labChanName">
                <property name="styleSheet">
                 <string notr="true">QLabel#labChanName {
font-weight: bold;
color: white;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frameNavi">
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="labCurrDay">
                <property name="styleSheet">
                 <string notr="true">QLabel#labCurrDay {
font-weight: bold;
color: white;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="CEpgBrowser" name="textEpg">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLayoutSearch">
          <property name="topMargin">
           <number>4</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labSearch">
            <property name="styleSheet">
             <string notr="true">QLabel#labSearch {
font-weight: bold;
color: white;
}</string>
            </property>
            <property name="text">
             <string>Search: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineSearch">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit#lineSearch {
  color: #800;   
  font: bold;
   background-image: url(:/app/search);
   background-repeat: no-repeat;
   background-position: LEFT;
   border: 1px inset;
   border-color: #555;
 }</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSearch">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="vlc-record.qrc">
              <normaloff>:/app/find</normaloff>:/app/find</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Ctrl+F</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame#frame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #eee, 	stop:0.7 grey, stop:1 #dedede);
background-image: url(:/app/time);
background-position: left center;
background-repeat: no-repeat;
background-attachment: fixed;
};</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>42</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QComboBox" name="cbxTimeShift">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set TimeShift in hours.</string>
        </property>
        <property name="whatsThis">
         <string>TimeShift Selector</string>
        </property>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushPlay">
        <property name="maximumSize">
         <size>
          <width>34</width>
          <height>34</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Play selected Channel</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="vlc-record.qrc">
          <normaloff>:/app/play</normaloff>:/app/play</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+P</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushRecord">
        <property name="maximumSize">
         <size>
          <width>34</width>
          <height>34</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Record selected Channel</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="vlc-record.qrc">
          <normaloff>:/app/record</normaloff>:/app/record</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+R</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushSettings">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Settings</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="vlc-record.qrc">
          <normaloff>:/app/settings</normaloff>:/app/settings</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+S</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushAbout">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>About ...</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="vlc-record.qrc">
          <normaloff>:/app/info</normaloff>:/app/info</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+I</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushQuit">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Quit</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="vlc-record.qrc">
          <normaloff>:/app/quit</normaloff>:/app/quit</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+Q</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CEpgBrowser</class>
   <extends>QTextBrowser</extends>
   <header>cepgbrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbxChannelGroup</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>textEpgShort</tabstop>
  <tabstop>cbxTimeShift</tabstop>
  <tabstop>pushPlay</tabstop>
  <tabstop>pushRecord</tabstop>
  <tabstop>textEpg</tabstop>
  <tabstop>lineSearch</tabstop>
  <tabstop>pushSettings</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>pushQuit</tabstop>
  <tabstop>pushAbout</tabstop>
 </tabstops>
 <resources>
  <include location="vlc-record.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushQuit</sender>
   <signal>clicked()</signal>
   <receiver>Recorder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
